metro.config.js

```diff
@@ -1,10 +1,22 @@
-module.exports = {
-    transformer: {
-        getTransformOptions: async () => ({
-            transform: {
-                experimentalImportSupport: false,
-                inlineRequires: false,
+const { getDefaultConfig } = require('metro-config');
+
+module.exports = (async () => {
+    const {
+        resolver: { sourceExts, assetExts },
+    } = await getDefaultConfig();
+        return {
+            transformer: {
+                babelTransformerPath: require.resolve('react-native-svg-transformer'),
+                getTransformOptions: async () => ({
+                    transform: {
+                        experimentalImportSupport: false,
+                        inlineRequires: false,
+                    },
+                }),
             },
-        }),
-    },
-};
+            resolver: {
+                assetExts: assetExts.filter(ext => ext !== 'svg'),
+                sourceExts: [...sourceExts, 'svg'],
+            },
+        };
```

Comment:
This block seems to have an extra indentation. Is this OK?

Reply:
Yeah, I don't know why. I ran ```lint -- --fix ``` but apparently with no success. This file is in eslint ignore. I'm remove that from there. I think we should have eslint on it

Reply:
I think it's OK to add a rule in the project itself to include this file for  the time being. When we develop our ESLint enhancer, this file will be included automatically! 💪